simple-example/{{cookiecutter.project_name}}/docker_build.sh

```diff
@@ -40,7 +50,23 @@ else
  echo "Registry set: creating tag ${TAG}"
 fi
 
-# Should be run in the folder that has Dockerfile
-docker build --tag ${TAG} .
+# convert *.ipynb to *.py
+if hash jupyter 2>/dev/null; then
+    echo "Jupyter found. converting workflows/*.ipython to *.py"
+    jupyter nbconvert workflows/*.ipynb --to script
+else
+    echo "Jupyter not found"
+fi
+
+# generate requirements.txt if pipenv were used
+if hash pipenv 2>/dev/null;then
+    echo "Pipenv found. Generating requirements.txt"
+    pipenv run pip freeze > requirements.txt
+else
+    echo "Pipenv not found"
+fi
```

Comment:
this kind of expands the scope and purpose of this `docker_build.sh` script to be opinionated about jupyter and pipenv as developer tools. I think this would make sense perhaps as part of steps a user manually runs in the `README.md` instructions.

Thoughts @eapolinario ?